pkg/logs/config/config.go

```diff
@@ -16,13 +16,16 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/logs/client"
 )
 
+// ContainerCollectAll is the name of the docker integration that collect logs from all containers
+const ContainerCollectAll = "container_collect_all"
+
 // DefaultSources returns the default log sources that can be directly set from the datadog.yaml or through environment variables.
 func DefaultSources() []*LogSource {
 	var sources []*LogSource
 
 	if coreConfig.Datadog.GetBool("logs_config.container_collect_all") {
```

Comment:
We just spent a significant amount of time trying to get this feature working. Nowhere [in the docs](https://docs.datadoghq.com/logs/log_collection/docker/?tab=environmentvariable) does it say you need to enable this in the datadog agent config (it only mentions by ENV var and labels) 😢

Reply:
@nunofgs thanks for reporting this, you're right we will fix this.